src/scss/components/wysiwyg/_table.scss

```diff
@@ -9,6 +9,8 @@
   }
 
   table {
+    @include modular-spacing('margin-top', 2);
```

Comment:
Looks good but modular-spacing is deprecated. Maybe replace it with @reponsive-spacing? 😃

Reply:
Actually I'm looking at the Decanter version in package.json and it's at 6.1.0 so this new mixin might not be in there. I'm going to approve it for now.